src/test/java/HomeWork/pages/DynamicControlsPage.java

```diff
@@ -0,0 +1,44 @@
+package HomeWork.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+
+
+public class DynamicControlsPage extends BasePage {
+    public final static By AFTER_LOADING_CHANGE = By.cssSelector("p[id=message]");
```

Comment:
It's not needed to use a Page Object pattern for HerokuApp tests. 
But if you use it - then please follow the pattern rules:
1. all locators should be incapsulated in page classes (should be private instead o public)
2. your tests should consist of page class methods and assertion. It should not be like this:
page method -> driver.find... -> wait.until...
<img width="940" alt="image" src="https://github.com/EvgeniyHotetovskiy/Herokuapp_QA26_Khotetovskiy/assets/14195681/517324ea-69a9-4b43-9c94-24961a767fee">